src/west/manifest.py

```diff
@@ -71,7 +297,7 @@ def manifest_path():
         raise OSError(errno.ENOENT, os.strerror(errno.ENOENT), ret)
     return ret
 
-def validate(data):
+def validate(data: Any) -> None:
     '''Validate manifest data
 
     Returns if the manifest data is valid and can be loaded by this
```

Comment:
Does not return, so comment should be in line with function.

Reply:
Sorry, I don't see the problem here. The comment is saying that the function returns if the manifest data is valid, which it does. It only raises an exception if the manifest data are invalid. Is there a bug in the docstring?

Reply:
If function returns something `-> None` is inconsistent.

Reply:
It either returns control flow back to the caller or raises an exception. If it returns, it returns None. So I think this is correct. No?

Reply:
The text says that returns if data is valid (suggesting a boolean return). In this case invalid means that exception is raised (so nothing is returned), so maybe `Raises exception if the manifest....` would be more explanatory.

Reply:
I see. The docstring is unclear. Here, "returns" is used intransitively. It doesn't return any value, it simply returns (at all). Thanks, will try to fix.